server/consumer.go

```diff
@@ -818,6 +823,9 @@ func checkConsumerCfg(
 			if group == _EMPTY_ {
 				return NewJSConsumerEmptyGroupNameError()
```

Comment:
Is this one used anywhere? Should we remove if not?

Reply:
you mean - remove the error, as the regex below will check for empty groups?

Reply:
ok if still used we are good, could not tell if the empty error was being used anymore.